css/modules/main-container-layout.css

```diff
@@ -335,13 +328,35 @@
     padding: 0.67857rem 0.8571rem 0.8571rem 0.9285rem;
   }
 
-  /* #virtualclassCont.congrea a.inner_bt {
-    padding: 0.6785rem 1rem 0.6428rem 1rem;
-  } */
+  html.pwaRunning #virtualclassCont.congrea #preCheckcontainer .modal {
+  background-color: #e4e4e4;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)